Bottom: 5d0f0deae6960b86ba3e8d74bae7fdb02fd995f6
Top:    0a9c637fd7176853cb898de9f4b782a21a02d500
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-26 00:14:05 +0100

Refresh of draw-c-split-gathering-routine

---

diff --git a/src/draw.c b/src/draw.c
index 5449a73..3d85e07 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1525,7 +1525,10 @@ DrawElementPinsAndPads (ElementTypePtr Element)
   PAD_LOOP (Element);
   {
     if (doing_pinout || doing_assy || FRONT (pad) || PCB->InvisibleObjectsOn)
-      DrawPad (pad);
+      if (Gathering)
+        DrawPad (pad);
+      else
+        draw_pad (pad);
   }
   END_LOOP;
   PIN_LOOP (Element);
